README.md

```diff
@@ -14,6 +14,8 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
 
 ## Contents
 
+- [Awesome Actions ![Awesome](https://github.com/sindresorhus/awesome)](#awesome-actions-awesomehttpsgithubcomsindresorhusawesome)
+- [Contents](#contents)
```

Comment:
@joelwmale can I ask why this was added?

Reply:
Will also revert this.